### PR TITLE
Update

### PR DESCRIPTION
**Pull request by Dqnns**
added some new [animals ](https://www.englishclub.com/vocabulary/animal-terms-easy.htm)& [fruits](https://www.halfyourplate.ca/fruits-and-veggies/fruits-a-z/).

### CODE REVIEW

**Comment by LabyStudio** [comment]
Did you remove Ponny and Turkey on purpose?

**Comment by Dqnns** [comment]
removed Ponny and added it below:

image

removed Turkey because i wanted to add a countries file in the future.

**Comment by Dqnns** [comment]
Sources: 

[](https://www.worldometers.info/geography/how-many-countries-in-africa/)
[](https://www.worldometers.info/population/countries-in-asia-by-population/)
[](https://www.worldometers.info/geography/how-many-countries-in-oceania/)
[](https://www.worldometers.info/geography/how-many-countries-in-europe/)
[](https://en.wikipedia.org/wiki/List_of_states_and_territories_of_the_United_States)
[](https://en.wikipedia.org/wiki/South_America)

**Comment by Dqnns** [comment]
[Sources] Countries

[Africa](https://www.worldometers.info/geography/how-many-countries-in-africa/)
[Asia](https://www.worldometers.info/population/countries-in-asia-by-population/)
[Australia - Oceania](https://www.worldometers.info/geography/how-many-countries-in-oceania/)
[Europe](https://www.worldometers.info/geography/how-many-countries-in-europe/)
[North America](https://en.wikipedia.org/wiki/List_of_states_and_territories_of_the_United_States)
[South America](https://en.wikipedia.org/wiki/South_America)

File: src/countries/Asia.txt [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+Lebanon
+Macao
+Malaysia
+Maldives
+Mongolia
+Nepal
+Northkorea
+Oman
+Pakistan
+Palestine
```
Comment (Dqnns) [inline src/countries/Asia.txt:28]: please don't argue on this.

**Comment by Dqnns** [comment]
I have not added every country, just the ones that everyone knows.

**Pull request merged**